### ISSUE TITLE
Clustering not working on signed apk.

### ISSUE THREAD

**Issue by vandarmetin1** [yandex/mapkit-android-demo#358]
Gradle plugin version: 8.1.1
TargetSdk: 33
CompileSdk: 34
Yandex Mapkit version: 4.3.1-full

When I run my project using Android Studio clustering works fine but when I generate signed apk and install it on my emulator, all the map object are disappearing when zooming out. I tried upgrading mapkit version to 4.4.0-full but it didn't work.

**Comment by vandarmetin1** [comment]
I have been trying to solve this for days but I did not find any solution :(

**Comment by YandexMapKit** [comment]
If you can, send us a minimal compiled example, that demonstrates the situation. It can be uploaded on [GitHub](https://github.com/), attached to mail as an archive, or uploaded to [Yandex Disk](https://disk.yandex.com/) and sent to us via link.

We will try to find a solution.

**Comment by vandarmetin1** [comment]
I trying to send you a compiled example rn.

**Comment by YandexMapKit** [comment]
Yes, [MapKit does keep weak references](https://yandex.com/maps-api/docs/mapkit/android/generated/getting_started.html#weak-references) to the Listener objects that are passed to it. However, you will need to keep references to these objects in your own code.
